es/lessons/basics/enum.md

```diff
@@ -68,27 +70,30 @@ Hay algunas opciones para `chunk_every/2` pero no vamos a entrar en detalle, rev
 
 ### chunk_by
 
-Si necesitas agrupar una colección basándose en algo diferente al tamaño, podemos usar la función `chunk_by`. Esta función recibe un enumerable como parámetro y una función, cada vez que la respuesta de esa función cambia un nuevo grupo es creado y continúa con la creación del siguiente:
+Si necesitamos agrupar una colección basándose en algo diferente al tamaño, podemos usar la función `chunk_by/2`.
+Esta función recibe un enumerable como parámetro y una función, cada vez que la respuesta de esa función cambia un nuevo grupo es creado y continúa con la creación del siguiente:
 
 ```elixir
 iex> Enum.chunk_by(["one", "two", "three", "four", "five"], fn(x) -> String.length(x) end)
 [["one", "two"], ["three"], ["four", "five"]]
-
 iex> Enum.chunk_by(["one", "two", "three", "four", "five", "six"], fn(x) -> String.length(x) end)
 [["one", "two"], ["three"], ["four", "five"], ["six"]]
 ```
+
 ### map_every
 
-Algunas veces hacer chunk_every/2 a una collección no es suficiente para lograr lo que necesitas. Si este es el caso, `map_every/3` puede ser muy útil para ejecutarce en elementos específicos si tu colección posee un orden en el que esto sea necesario o útil':
+Algunas veces hacer chunk a una collección no es suficiente para lograr lo que necesitas.
```

Comment:
`collección` -> `colección`

Reply:
me paso de nuevo :(

Reply:
I had some merge issue, it should be OK, now